packages/open-next/src/core/routing/matcher.ts

```diff
@@ -216,7 +216,7 @@ export function handleRewrites<T extends RewriteDefinition>(
       rewrittenQuery = unescapeRegex(toDestinationQuery(params));
     }
     rewrittenUrl = isExternalRewrite
-      ? `${protocol}//${rewrittenHost}${rewrittenPath}`
```

Comment:
Thanks! Just to be overly cautious due to nextjs sometimes being inconsistent... are there scenarios where `protocol` doesn't actually include the `//`?

Reply:
The `protocol` is returned by the utils function `getUrlParts` which always ensures that `https://` is returned

https://github.com/sst/open-next/blob/9d632d1bd5c3a8d7e7bbd13ae9daaca68da9284e/packages/open-next/src/core/routing/util.ts#L43-L66

Reply:
Usually the protocol doesn't include the `//`, so this function should probably be updated.
https://developer.mozilla.org/en-US/docs/Web/API/URL/protocol

Reply:
Ha valid point indeed! I'll adapt